x-pack/legacy/plugins/siem/public/components/ml/tables/network_equality.test.ts

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { dateTimesAreEqual } from './date_time_equality';
+import { AnomaliesNetworkTableProps } from '../types';
+import { NetworkType } from '../../../store/network/model';
+import { FlowTarget } from '../../../graphql/types';
+
+describe('network_equality', () => {
+  test('it returns true if start and end date are equal', () => {
+    const prev: AnomaliesNetworkTableProps = {
+      startDate: new Date('2000').valueOf(),
+      endDate: new Date('2000').valueOf(),
+      narrowDateRange: jest.fn(),
+      skip: false,
+      type: NetworkType.details,
+    };
+    const next: AnomaliesNetworkTableProps = {
+      startDate: new Date('2000').valueOf(),
+      endDate: new Date('2000').valueOf(),
+      narrowDateRange: jest.fn(),
+      skip: false,
+      type: NetworkType.details,
+    };
+    const equal = dateTimesAreEqual(prev, next);
+    expect(equal).toEqual(true);
+  });
+
+  test('it returns false if starts are not equal', () => {
+    const prev: AnomaliesNetworkTableProps = {
+      startDate: new Date('2000').valueOf(),
+      endDate: new Date('1999').valueOf(),
+      narrowDateRange: jest.fn(),
+      skip: false,
+      type: NetworkType.details,
+    };
+    const next: AnomaliesNetworkTableProps = {
+      startDate: new Date('2000').valueOf(),
+      endDate: new Date('2000').valueOf(),
+      narrowDateRange: jest.fn(),
+      skip: false,
+      type: NetworkType.details,
+    };
+    const equal = dateTimesAreEqual(prev, next);
+    expect(equal).toEqual(false);
+  });
+
+  test('it returns false if starts are not equal for next', () => {
+    const prev: AnomaliesNetworkTableProps = {
+      startDate: new Date('2000').valueOf(),
+      endDate: new Date('2000').valueOf(),
+      narrowDateRange: jest.fn(),
+      skip: false,
+      type: NetworkType.details,
+    };
+    const next: AnomaliesNetworkTableProps = {
+      startDate: new Date('2000').valueOf(),
+      endDate: new Date('1999').valueOf(),
```

Comment:
As with the `host_equality` tests above, looks like you meant to have the difference be the `startDate` here.